newsrevenuehub/rev-engine
# [NRH-239] Footer-esque content

#### What's this PR do?
Adds a donation page footer with link to fundjournalism.org and copyright.
Adds main-content footer with dynamic text based on org values and amount/frequency.

#### How should this be manually tested?
Visit a live donation page and observe the existence of a footer with the appropriate content. Adjust amount and frequency and see the other footer-type content change. Make changes to the organization contact email, organization address, and non-profit status to observe other changes to the text.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
